responses.go

```diff
@@ -211,8 +228,8 @@ type ProducerChange struct {
 type AccountResp struct {
 	AccountName            AccountName          `json:"account_name"`
 	Privileged             bool                 `json:"privileged"`
-	LastCodeUpdate         JSONTime             `json:"last_code_update"`
-	Created                JSONTime             `json:"created"`
+	LastCodeUpdate         BlockTimestamp       `json:"last_code_update"` // modified 2022-11-02
+	Created                BlockTimestamp       `json:"created"`          // modified 2022-11-02
```

Comment:
What the reasoning of those changes?

Reply:
https://github.com/psy2848048/eos-go/blob/b489eeb186b3aa46096c1deff3f75ed651d6ebf8/testdata/mock_server/chain_get_account.json#L6

Reply:
https://github.com/eoscanada/eos-go/pull/191/commits/18d78b47dd7be8c506ce92f5422a4273eb0d1820

Reply:
https://github.com/psy2848048/eos-go/blob/b489eeb186b3aa46096c1deff3f75ed651d6ebf8/testdata/mock_server/chain_get_account.json#L6